web_widget_date_interval/static/src/js/web_widget_date_interval.js

```diff
@@ -75,17 +75,22 @@ openerp.web_widget_date_interval = function (instance) {
                 }
                 return result;
             },
+            weeknumber_iso_date_from_week: function (year, week) {
+                return new Date(year, 5, 1).setWeek(week);
+            },
             weeknumber_iso_onchange_year: function (e) {
-                var year = parseInt(jQuery(e.currentTarget).data('year'), 10),
-                    value = this._get_date();
-                value.setFullYear(year);
-                return this.weeknumber_iso_update_interval(value);
+                var date = this._get_date();
+                var year = parseInt(jQuery(e.currentTarget).data('year'), 10);
+                var week = this.weeknumber_iso_weeknumber(date);
+                var monday = this.weeknumber_iso_date_from_week(year, week);
```

Comment:
how are we sure this is a monday? maybe better set it like that in `weeknumber_iso_date_from_week`

Reply:
To my understanding [this](https://github.com/datejs/Datejs/blob/master/src/core.js#L410) always returns a monday